tests/StorageBackendTest.php

```diff
@@ -147,12 +159,24 @@ public function testUpdateStorageBackend(array $options)
             'password' => getenv('KBC_TEST_SNOWFLAKE_BACKEND_PASSWORD'),
         ];
```

Comment:
BTW tohle je tady zbytecny, kdyz uz je to obsazeny v `$options`, ne?

Reply:
No, ono je to pitomě napsané. Ono to dostane komplet v $options. Udělá to z toho novej backend. A pak to přepíše a v options je jen heslo a tím se to updatne na ř. 168

Reply:
jezisi jo, sorry, uz to chapu 🤦  Jsem si myslel, ze to updatuje. OK